orchestra/migrations/0083_workflowversion_abort_completion_function.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.9 on 2020-04-09 14:27
```

Comment:
we've already got an 0083 :) https://github.com/b12io/orchestra/blob/master/orchestra/migrations/0083_update_staffbot_request_status_choices.py